packages/features/bookings/Booker/components/hooks/useInitialFormValues.ts

```diff
@@ -96,25 +103,30 @@ export function useInitialFormValues({
         view: rescheduleUid ? "reschedule" : "booking",
       });
 
-      const parsedQuery = await querySchema.parseAsync({
-        ...extraOptions,
-        name: prefillFormParams.name,
-        // `guest` because we need to support legacy URL with `guest` query param support
-        // `guests` because the `name` of the corresponding bookingField is `guests`
-        guests: prefillFormParams.guests,
-      });
+      // If organization has disabled autofill, don't use URL parameters for prefill
+      const urlParamsToUse = isAutofillDisabledByOrg
+        ? {}
+        : {
+            ...extraOptions,
+            name: prefillFormParams.name,
+            // `guest` because we need to support legacy URL with `guest` query param support
+            // `guests` because the `name` of the corresponding bookingField is `guests`
+            guests: prefillFormParams.guests,
+          };
+
+      const parsedQuery = await querySchema.parseAsync(urlParamsToUse);
 
       const defaultUserValues = {
         email:
           rescheduleUid && bookingData && bookingData.attendees.length > 0
             ? bookingData?.attendees[0].email
-            : !!parsedQuery["email"]
+            : parsedQuery["email"]
             ? parsedQuery["email"]
             : email ?? "",
```

Comment:
We need to disable any kind of auto-fill as part of this feature. So, prefill/auto fill from session info should also be disabled.

The only prefill that makes sense is the booking details prefill itself which is there during reschedule. We can't avoid that.